CONTRIBUTING.md

```diff
@@ -1,80 +1,233 @@
+  scenarios are more interesting than rare ones. RFCs should do more than offer
+  a different way to reach an already achievable outcome.
 
-**Steps**: Merge graphql-js PR and release NPM package
+* **Simplicity and consistency over expressiveness and terseness**
 
-**Post-Acceptance Changes Expected**: Only those deemed critical based on
-implementation experience.
+  There are plenty of behaviors and patterns found in other languages
+  intentionally absent from GraphQL. "Possible but awkward" is often favored
+  over more complex alternatives. Simplicity (e.g. fewer concepts) is
+  more important than expressing more sophisticated ideas or writing less.
 
-## Stage 3: Merged
+* **Preserve option value**
 
-**Prerequisite**:
-+ At least one month since release of the graphql-js with proposed change.
-+ Community consent on proposed change. If it’s hard to achieve add to agenda of
-the next WG meeting..
+  It's hard to know what the future brings, so whenever possible decisions
+  should be made which allow for more options in the future. Sometimes this is
+  unintuitive: spec rules often begin more strict than necessary with a future
+  option to loosen when motivated by a real use case.
 
-**Steps**: Merge PR into “master” branch.
+* **Understandability is just as important as correctness**
+  The GraphQL spec, despite describing technical behavior, is intended to be
+  read by people. Use natural tone and include motivation and examples.
 
-**Post-Acceptance Changes Expected**: None.
+
+## RFC Contribution Champions
+
+Contributing to GraphQL requires a lot of dedicated work. To set clear
+expectations and provide accountability, each proposed RFC (request for
+comments) must have a *champion* who is responsible for addressing feedback and
+completing next steps. An RFC may have multiple *champions*. The spec editors
+are not responsible for completing RFCs which lack a *champion* (though an
+editor may be a *champion* for an RFC).
+
+An RFC which does not have a *champion* may not progress through stages, and can
+become stale. Stale proposals may be picked up by a new *champion* or may
+be *rejected*.
+
+
+## RFC Contribution Stages
+
+RFCs are guided by a *champion* through a series of stages: *strawman*,
+*proposal*, *draft*, and *accepted* (or *rejected*), each of which has suggested
+entrance criteria and next steps detailed below. RFCs typically advance one
+stage at a time, however may advance multiple stages at a time. Stage
+advancements typically occur during
+[Working Group](https://github.com/graphql/graphql-wg) meetings, however may
+occur online.
+
+All RFCs start as either a *strawman* or *proposal*.
+
+## Stage 0: *Strawman*
+
+The *strawman* stage captures described problems or partially-considered
+solutions which lack the entrance criteria to be considered a *proposal*. A
+*strawman* may be an issue or a pull request (though an illustrative pull
+request is preferrable).
```

Comment:
Thesis: RFC with goal of comments
Requirements: very little
Why: To bring attention to issues and/or seek advice on solutions.
```
A *strawman* RFC captures described problems or partially-considered
solutions. The *strawman* does not need to meet the *proposal* entrance
criteria. A *strawman* may be an issue or a pull request (though an
illustrative pull request is preferrable).
```